build.cake

```diff
@@ -66,38 +62,6 @@ var packageWhitelist = new[] { "ReactiveUI-Testing", "ReactiveUI-Events", "React
 // Define global marcos.
 Action Abort = () => { throw new Exception("a non-recoverable fatal error occurred."); };
 
-Action<string> RestorePackages = (solution) =>
-{
-    NuGetRestore(solution, new NuGetRestoreSettings() { ConfigFile = "./src/.nuget/NuGet.config" });
-};
-
-Action<string, string> Package = (nuspec, basePath) =>
-{
-    CreateDirectory(artifactDirectory);
-
-    Information("Packaging {0} using {1} as the BasePath.", nuspec, basePath);
-
-    NuGetPack(nuspec, new NuGetPackSettings {
-        Authors                  = new [] {"ReactiveUI contributors"},
-        Owners                   = new [] {"xpaulbettsx", "flagbug", "ghuntley", "haacked", "kent.boogaart", "mteper", "moswald", "niik", "onovotny", "rdavisau", "shiftkey"},
-
-        ProjectUrl               = new Uri("http://www.reactiveui.net"),
-        IconUrl                  = new Uri("https://i.imgur.com/7WDbqSy.png"),
-        LicenseUrl               = new Uri("https://opensource.org/licenses/ms-pl.html"),
-        Copyright                = "Copyright (c) ReactiveUI and contributors",
-        RequireLicenseAcceptance = false,
-
-        Version                  = nugetVersion,
-        Tags                     = new [] {"mvvm", "reactiveui", "Rx", "Reactive Extensions", "Observable", "LINQ", "Events", "xamarin", "android", "ios", "forms", "monodroid", "monotouch", "xamarin.android", "xamarin.ios", "xamarin.forms", "wpf", "winforms", "uwp", "winrt", "net45", "netcore", "wp", "wpdev", "windowsphone", "windowsstore"},
-        ReleaseNotes             = new [] { string.Format("{0}/releases", githubUrl) },
-
-        Symbols                  = false,
-        Verbosity                = NuGetVerbosity.Detailed,
-        OutputDirectory          = artifactDirectory,
-        BasePath                 = basePath,
-    });
-};
-
 Action<string> SourceLink = (solutionFileName) =>
 {
     GitLink("./", new GitLinkSettings() {
```

Comment:
Oops, missed that one then amongst the other errors. Will take a look at that, at least some of the .NetStandard projects have it behaving.